.travis.yml

```diff
@@ -6,8 +6,10 @@ node_js:
 env:
   - FORMDATA_VERSION=1.0.0
   - FORMDATA_VERSION=2.1.0
+  - ENCODING=yes
 before_script:
   - 'if [ "$FORMDATA_VERSION" ]; then npm install form-data@^$FORMDATA_VERSION; fi'
+  - 'if [ "$ENCODING" = "yes" ]; then npm install encoding; fi'
```

Comment:
Is this an okay way to add that env var in travis? This way doesn't seem to matrix each possibility (each version of form-data with and without encoding).. is that important to test or you think no?